src/shared/credentials/userCredentialsUtils.ts

```diff
@@ -194,12 +199,8 @@ export class UserCredentialsUtils {
                 return true
             }
         } catch (err) {
-            // swallow any errors--anything that isn't a success should end in the same state:
-            // no set profile + return false
+            // swallow any errors--anything that isn't a success be handled as a failure by the caller
```

Comment:
Nit: Missing a word ("*should* be handled"?)

Reply:
✔️